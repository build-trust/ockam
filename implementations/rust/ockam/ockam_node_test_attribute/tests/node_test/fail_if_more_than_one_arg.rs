// Test case to verify that only one argument is passed.

#[ockam_node_test_attribute::node_test]
async fn my_test(mut c: ockam::Context, _x: u64) {
    c.stop().await.unwrap();
}
