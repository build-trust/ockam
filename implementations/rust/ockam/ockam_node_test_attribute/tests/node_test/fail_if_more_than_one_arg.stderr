error: the function must have exactly one argument
 --> $DIR/fail_if_more_than_one_arg.rs:4:7
  |
4 | async fn my_test(mut c: ockam::Context, _x: u64) -> ockam::Result<()> {
  |       ^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> $DIR/fail_if_more_than_one_arg.rs:3:1
  |
3 | / #[ockam_node_test_attribute::node_test]
4 | | async fn my_test(mut c: ockam::Context, _x: u64) -> ockam::Result<()> {
5 | |     c.stop().await.unwrap();
6 | | }
  | |_^ consider adding a `main` function to `$DIR/tests/node_test/fail_if_more_than_one_arg.rs`
